packages/GA4Client/src/initialization.js

```diff
@@ -30,6 +30,19 @@ var initialization = {
         window.gtag = function () {
             window.dataLayer.push(arguments);
         };
+        gtag('js', new Date());
+
+        if (window.mParticle.getVersion().split('.')[0] === '2') {
```

Comment:
Please clarify why you are doing string manipulation here. I would recommend a convenience method called `isV2` or add a comment explaining why we are checking the version number. I would prefer code as a comment, but either one would work.